### DISCUSSION TITLE
Donwload  von PDFS  mit Begleitmaterial.

### DISCUSSION THREAD

**Discussion by alurio** [audiamus/BookLibConnect#26] [Ideas]
Hallo Audiamus, 

Manchmal bietet Audible ja zu Hörbüchern Begleitmaterial als PDF an. Ist geplant dass diese von Booklibconnect oder später einmal von AAX Audioconverter 2 heruntergeladen werden? Hat jetzt nicht die höchste Dringlichkeit, weil man kann sich die PDFs ja einfach manuell aus der Bibliothek laden, aber ein automatisches herunterladen  wäre ja zumindest praktisch. 

Viele Grüße

**Comment by audiamus** [comment]
Ja, steht auf der Liste, aber nicht mit hoher Priorität. 

Das Verfahren, nach dem Audible solche PDF-Zusätze zur Verfügung stellt, ist mir allerdings schleierhaft. Ich habe einige wenige Hörbücher auch in gedruckter Form. Während die gedruckten Ausgaben teils einzelne Karten, Skizzen, Illustrationen oder Fotos enthalten, vielleicht sogar essentieller Art, war beim Hörbuch kein PDF dabei.
